### PR TITLE
Add imroc as a member

### PR DESCRIPTION
**Pull request by imroc**
The most common reason to send a PR to this repository is to join the Istio org.  If you are joining, please fill out this template.  If you are not, please **delete all the below**.

- [x] I have reviewed the [community membership guidelines](https://github.com/istio/community/blob/master/ROLES.md#member)
- [x] I have reviewed the [contribution guidelines](https://github.com/istio/community/blob/master/CONTRIBUTING.md)
- [x] I have enabled [2FA on my GitHub account](https://github.com/settings/security)
- [x] I have joined the [Istio discussion board](https://discuss.istio.io) and subscribed to the [Contributors](https://discuss.istio.io/c/contributors) category
- [x] I have joined [Istio's Slack workspace](https://slack.istio.io)

List your company name, or indicate Individual if you're not affiliated with a company:
Tencent

Provide a link to at least one PR that you have successfully pushed to one of the Istio repos:
https://github.com/istio/istio/pull/41203

### CODE REVIEW

**Review by Xunzhuo — APPROVED** [review]

**Review by kebe7jun — APPROVED** [review]
Welcome!

File: org/members.yaml [hunk 1]
```diff
@@ -388,3 +388,4 @@ members:
 - zqzzq
 - zufardhiyaulhaq
 - zyfjeff
+- imroc
```
Comment (kebe7jun) [inline org/members.yaml:391]: needs sort list.

**Review by kebe7jun — CHANGES_REQUESTED** [review]

**Review by hzxuzhonghu — APPROVED** [review]

**Review by kebe7jun — APPROVED** [review]

**Comment by imroc** [comment]
@kebe7jun 
Sorry I didn't notice the email before, the invitation is out of date, can you resend the invitation?

**Comment by kebe7jun** [comment]
> @kebe7jun Sorry I didn't notice the email before, the invitation is out of date, can you resend the invitation?

I'm not familiar with how to do this, @craigbox  do you have experience to help imroc?

**Comment by imroc** [comment]
> @kebe7jun Sorry I didn't notice the email before, the invitation is out of date, can you resend the invitation?

@howardjohn @dhawton Does anyone know how to do it? It is a bit troublesome to ask other members to "/ok-to-test" for each PR

**Comment by dhawton** [comment]
I do not sorry.

**Comment by ericvn** [comment]
Looking at the post submit job, it simply says it waiting for an acceptance. If I look at the member list in GitHub, I see an entry under pending invitations, and I only see the ability to edit an invite and cancel an invite, not to resend an invite. I did select to edit the invite, did not change it, and clicked to update. I'm not sure if that will send another invite or not. We could also cancel the invite, and then when the next community PR merges, the post-submit should then create a new invite (may need the PR to have a change in the members org to cause the job to run (would need to double-check)).

**Comment by imroc** [comment]
> Looking at the post submit job, it simply says it waiting for an acceptance. If I look at the member list in GitHub, I see an entry under pending invitations, and I only see the ability to edit an invite and cancel an invite, not to resend an invite. I did select to edit the invite, did not change it, and clicked to update. I'm not sure if that will send another invite or not. We could also cancel the invite, and then when the next community PR merges, the post-submit should then create a new invite (may need the PR to have a change in the members org to cause the job to run (would need to double-check)).

@ericvn It works, thank you so much.

**Pull request merged**